app/buck2_client/src/commands/test.rs

```diff
@@ -225,6 +255,8 @@ impl StreamingCommand for TestCommand {
                         .transpose()
                         .context("Invalid `timeout`")?,
                     ignore_tests_attribute: self.ignore_tests_attribute,
+                    build_default_info: !self.skip_default_info,
+                    build_run_info: !self.skip_run_info,
```

Comment:
Isn't this missing a `|| self.build_default_info`?

Reply:
Not really because the two flags are exclusive by being in the same group, so only checking the opt-in is enough. This is how it is done for the `build` command, so I kept things consistent, but I don't mind being explicit instead if you prefer.

Reply:
Right, but the opt-in is `--build-default-info`, no? At least that's what the comment claims

Reply:
Of course, yes :facepalm:
Sorry for having you double check my logic, I will address this tomorrow.